main_test.go

```diff
@@ -3,152 +3,209 @@ package main
 import (
 	"os"
 	"testing"
-
-	"github.com/atotto/clipboard"
+	"github.com/xyproto/randomstring"
 )
 
-func TestCopyToClipboard(t *testing.T) {
-	if testing.Short() {
-		t.Skip("skipping clipboard test in CI/CD.")
+// TestParseContent_MultipleFiles verifies parseContent correctly concatenates contents of multiple files.
+func TestParseContent_MultipleFiles(t *testing.T) {
+	// Prepare test data
+	content1 := "Content from file 1"
+	content2 := "Content from file 2"
+	file1 := createTempFile(t, content1)
+	defer os.Remove(file1.Name())
+	file2 := createTempFile(t, content2)
+	defer os.Remove(file2.Name())
+
+	// Expected result after parsing multiple files
+	expected := "Content from file 1\nContent from file 2\n"
+
+	// Execute the function under test
+	actual, err := parseContent(nil, []string{file1.Name(), file2.Name()})
```

Comment:
If you do this, please consider using a variadic in parseContent

This way, you will use a his here

```suggestion
	actual, err := parseContent(nil, file1.Name(), file2.Name())
```

By variadic I mean this

```diff
-func parseContent(directText *string, args []string) (string, error) {
+func parseContent(directText *string, args ...string) (string, error) {
```

Reply:
I'm surprised by the nil passed to parseContent 🤔

If it's normal, please consider adding a variable with a meaningful name, set it to nil, and pass it to parseContent

Reply:
> If you do this, please consider using a variadic in parseContent
> 
> This way, you will use a his here
> 
> By variadic I mean this
> 
> ```diff
> -func parseContent(directText *string, args []string) (string, error) {
> +func parseContent(directText *string, args ...string) (string, error) {
> ```

The current integration with flag.Args() is working as expected and there's no strong reason to change it, it's generally advisable to keep the parseContent function signature consistent with what flag.Args() returns ([]string). Introducing a variadic parameter (args ...string) in the parseContent function would change the way arguments are passed from flag.Args() in the main function.

Reply:
Indeed it makes sense, thanks for replying